Doc/library/io.rst

```diff
@@ -600,9 +600,8 @@ Raw File I/O
    .. versionchanged:: 3.4
       The file is now non-inheritable.
 
-   In addition to the attributes and methods from :class:`IOBase` and
-   :class:`RawIOBase`, :class:`FileIO` provides the following data
-   attributes:
+   :class:`FileIO` provides these data attributes in addition to those from
```

Comment:
```suggestion
   :class:`FileIO` provides these attributes in addition to those from
```

Reply:
Alright, but you have forgotten "data".

Reply:
I disagree here, why removing "data"? There is no method involved here.

Reply:
Makes sense, thanks for linking to that.